bhs_api/fsearch.py

```diff
@@ -221,14 +222,11 @@ def clean_person(person):
             pass
 
     # remove the details of the living
-    if 'birth_year' in person and isinstance(person['birth_year'], int) and \
-       datetime.now().year - int(person['birth_year']) < 100 or \
-       'deceased' in person and \
-       not person['deceased']:
+    if is_living_person(person.get('deceased'), person.get('birth_year')):
         for key in person.keys():
             if key in ['birth_year', 'death_year', 'birth_place',
                        'death_place', 'marriage_place', 'marriage_date',
-                       'occupation', 'bio',
+                       'occupation', 'bio', 'BIRT_DATE'
```

Comment:
there are many more fields to clean. check out the forntend template [here](https://github.com/Beit-Hatfutsot/dbs-front/blob/dev/templates/main/ftrees/node-details.html#L39)